tests/integration/itest_posts.py

```diff
@@ -1,61 +1,56 @@
 import pytest
-from aleph_message import Message
-from aleph_message.models import MessagesResponse
-
+from aleph_message.models import PostMessage
+from aleph.sdk.exceptions import MessageNotFoundError
 from tests.integration.toolkit import try_until
-
 from .config import REFERENCE_NODE, TARGET_NODE
+from aleph.sdk import AuthenticatedAlephClient
+from aleph.sdk.conf import settings as sdk_settings
+from aleph.sdk import AlephClient
+
+
+async def get_message(item_hash: str):
+    async with AlephClient(api_server=sdk_settings.API_HOST) as client:
+        try:
+            response = await client.get_message(item_hash, message_type=PostMessage)
+            return response
+        except MessageNotFoundError:
+            return None
 
 
 async def create_message_on_target(
-    fixture_account, emitter_node: str, receiver_node: str
+        fixture_account, emitter_node: str, receiver_node: str
 ):
     """
     Create a POST message on the target node, then fetch it from the reference node.
     """
-
-    post_message, message_status = await create_post(
-        account=fixture_account,
-        post_content=None,
-        post_type="POST",
-        channel="INTEGRATION_TESTS",
-        session=None,
-        api_server=emitter_node,
-    )
-
-    def response_contains_messages(response: MessagesResponse) -> bool:
-        return len(response.messages) > 0
-
-    # create_message = Message(**created_message_dict)
-    response_dict = await try_until(
-        get_messages,
-        response_contains_messages,
-        timeout=5,
-        hashes=[post_message.item_hash],
-        api_server=receiver_node,
+    data = {'content': 'test'}
+    async with AuthenticatedAlephClient(
+            account=fixture_account, api_server=sdk_settings.API_HOST
+    ) as client:
+        result, status = await client.create_post(
```

Comment:
Rename result to message.

Reply:
Also, now that the SDK returns a status, you can assert that the operation worked with `assert status == ...`.